src/site/content/en/blog/idle-detection/index.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Detecting when the user is idle
+subhead: Use the Idle Detection API to find out when the user isn't actively using their device.
+authors:
+  - thomassteiner
+description: |
+  The Idle Detection API notifies developers when a user is idle, indicating such things as lack of
+  interaction with the keyboard, mouse, screen, activation of a screensaver, locking of the screen,
+  or moving to a different screen. A developer-defined threshold triggers the notification.
+date: 2020-04-20
+tags:
+  - post
+  - idle-detection
+  - fugu
+  - capabilities
+hero: hero.jpg #https://images.unsplash.com/photo-1544239265-ee5eedde5469?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1900&q=80
+alt: Abandoned computer on a bed with someone's leg next to it.
+draft: true
+origin_trial:
+  url: TODO
```

Comment:
TODO

Reply:
The final ToDo closes with the link being https://developers.chrome.com/origintrials/#/view_trial/551690954352885761.